src/libs/OptionsListUtils.js

```diff
@@ -161,36 +161,34 @@ function getAvatarsForLogins(logins, personalDetails) {
 }
 
 /**
- * Returns the personal details for an array of logins
+ * Returns the personal details for an array of accountIDs
  *
- * @param {Array} logins
+ * @param {Array} accountIDs
  * @param {Object} personalDetails
  * @returns {Object} – keys of the object are emails, values are PersonalDetails objects.
  */
-function getPersonalDetailsForLogins(logins, personalDetails) {
+function getPersonalDetailsForAccountIDs(accountIDs, personalDetails) {
     const personalDetailsForLogins = {};
     if (!personalDetails) {
         return personalDetailsForLogins;
     }
-    _.chain(logins)
-
-        // Somehow it's possible for the logins coming from report.participants to contain undefined values so we use compact to remove them.
+    _.chain(accountIDs)
+        // Somehow it's possible for the accountIDs coming from report.participantAccountIDs to contain undefined values so we use compact to remove them.
         .compact()
-        .each((login) => {
-            let personalDetail = personalDetails[login];
+        .each((accountID) => {
+            let personalDetail = personalDetails[accountID];
             if (!personalDetail) {
                 personalDetail = {
-                    login,
-                    displayName: LocalePhoneNumber.formatPhoneNumber(login),
-                    avatar: UserUtils.getDefaultAvatar(login),
+                    accountID,
```

Comment:
One thing @puneetlath and I are thinking is: If we don't have a display name to show we maybbbe want to just show "hidden" everywhere, OR we want to hide this - this is something we'll have to keep in mind when doing final testing

Reply:
Ah, have the displayName be 'hidden'? Do we want to change this right now, or is it something still being discussed?

Reply:
Well we will plan to show "hidden" on the the profile page that we're updating (PR: https://github.com/Expensify/App/pull/20144) - IFF you don't have that person's personal details in Onyx. So we were just "thinking" this may apply to other pages in the app, but I'd say let's just see what happens as we go - I think it would be better if we try to figure out "why" personal details may or may not be available in this scenario, and handle it from there, not just add "hidden" everywhere this happens and gloss over the _why_ 😅